### ISSUE TITLE
Support other image attributes

### ISSUE THREAD

**Issue by contra** [jessy1092/react-semantify#39]
Image seems to be missing circular, aligned, rounded, and a bunch of other classes

http://semantic-ui.com/elements/image.html

**Comment by jessy1092** [comment]
@contra Yes, I didn't implement `Variations` attributes because each component has different value. It cannot just use the common mixin function. Maybe it should implement one by one. Any idea?
